Library/Formula/rethinkdb.rb

```diff
@@ -32,6 +32,11 @@ def install
     system "make", "install-osx"
 
     (var/"log/rethinkdb").mkpath
+
+    inreplace "packaging/assets/config/default.conf.sample",
+              /^# directory=.*/, "directory=#{var}/rethinkdb"
+    cp "packaging/assets/config/default.conf.sample", "rethinkdb.conf"
```

Comment:
Can simplify this further with `etc.install "packaging/assets/config/default.conf.sample" => "rethinkdb.conf"`